containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml

```diff
@@ -121,18 +126,19 @@ openstack keypair create --public-key /home/jenkins-slave/.ssh/id_rsa.pub $BUILD
 trap "openstack keypair delete $BUILD_TAG" EXIT
 
 rm -rf $WORKSPACE/adt-run-output
-adt-run .// --setup-commands setup-testbed -o $WORKSPACE/adt-run-output --- ssh -d -l ubuntu -o='-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no' -s nova -- -d --flavor autopkgtest --image 'ubuntu/ubuntu-xenial-16.04-amd64-server-*' --keyname $BUILD_TAG --net-id=net_ues-snappy-integration --security-groups ssh -e 'http_proxy=http://squid.internal:3128' -e 'https_proxy=http://squid.internal:3128' -e 'no_proxy=127.0.0.1,127.0.1.1,localhost,localdomain,novalocal,internal,archive.ubuntu.com,security.ubuntu.com,ddebs.ubuntu.com,ppa.launchpad.net'
+adt-run .// --setup-commands setup-testbed -o $WORKSPACE/adt-run-output --- ssh -d -l ubuntu -o=&apos;-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no&apos; -s nova -- -d --flavor autopkgtest --image &apos;ubuntu/ubuntu-xenial-16.04-amd64-server-*&apos; --keyname $BUILD_TAG --net-id=net_ues-snappy-integration --security-groups ssh -e &apos;http_proxy=http://squid.internal:3128&apos; -e &apos;https_proxy=http://squid.internal:3128&apos; -e &apos;no_proxy=127.0.0.1,127.0.1.1,localhost,localdomain,novalocal,internal,archive.ubuntu.com,security.ubuntu.com,ddebs.ubuntu.com,ppa.launchpad.net&apos;
+
+subunit2junitxml $WORKSPACE/adt-run-output/artifacts/results.subunit -o $WORKSPACE/adt-run-output/artifacts/results.xml
 ]]></command>
     </hudson.tasks.Shell>
   </builders>
   <publishers>
-    <hudson.tasks.ArtifactArchiver>
-      <artifacts>adt-run-output/binaries/*.deb</artifacts>
```

Comment:
Not sure if we are going to have problems with the extended storage needed, anyway there are more people asking for it, i'll reenable them